### PR TITLE
W-16175211 RPA Manager release notes 2024-07/1

### PR DESCRIPTION
**Pull request by bookwoody**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,31 @@
...
+// == What's New
+//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Invocable run configuration: the assigned multi sessions are not correctly saved and not able to be updated | W-16169918
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| RPA Manager now properly saves the changes when you assign multiple sessions to an invocable run configuration. | W-16169918
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[july-05-2024]]
+== July 05, 2024
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
[[july-5-2024]]
== July 5, 2024
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,31 @@
 
 RPA Manager is the cloud-based control plane for your automation projects.
 
+[[july-5-2024]]
+== July 5, 2024
+
+// == What's New
+//include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]
+
```
Comment (bjoson-mulesoft) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:10]: Do we now include the What's New heading even if there aren't any new features? I haven't done this in the past, and don't see the guidance for that in the standards page, although I could be missing it. 
https://confluence.internal.salesforce.com/display/MTDT/Fixed+Issue+Descriptions

**Review by bjoson-mulesoft — APPROVED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc (hunk 3 above)
Comment (Cristian-Venticinque) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc:10]: The header is commented out, so it won't show.

**Pull request merged**